src/main/java/org/gridsuite/shortcircuit/server/reports/ReportMapperShortCircuit.java

```diff
@@ -44,11 +45,14 @@ public class ReportMapperShortCircuit extends AbstractReportMapper {
      * @implNote we assume there will always be at least one modification
      */
     @Override
-    protected ReportNode forShortCircuitAnalysis(@NonNull final ReportNode reportNode) {
+    protected ReportNode forShortCircuitAnalysis(@NonNull final ReportNode reportNode, ShortCircuitRunContext runContext) {
         ReportNodeBuilder builder = ReportNode.newRootReportNode()
                 .withMessageTemplate(reportNode.getMessageKey(), reportNode.getMessageTemplate());
         reportNode.getValues().entrySet().forEach(entry -> builder.withTypedValue(entry.getKey(), entry.getValue().getValue().toString(), entry.getValue().getType()));
         final ReportNode newReporter = builder.build();
+        if (!runContext.getInconsistentVoltageLevels().isEmpty()) {
```

Comment:
This condition should be before the run of the computation. If true, we store a failed status, an empty result and the reports that you build in forVoltageLevelsWithWrongIpValues